bool set_state
bool target_state
---
bool current_state
string message